src/style.css

```diff
@@ -0,0 +1,55 @@
+body {
+  margin: 0;
+}
+
+.container {
+  height: 96px;
+  width: 96px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  background-image: url(images/star.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+}
+
+.stars--1 .stars_star:first-child {
+  background-image: url(images/star-active.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
```

Comment:
these styles are already set for `.stars_star` class so there is no need to duplicate